Vagrantfile

```diff
@@ -5,7 +5,7 @@
 VAGRANTFILE_API_VERSION = "2"
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  config.vm.box = "chef/centos-6.5"
+  config.vm.box = "puppetlabs/centos-6.6-64-puppet"
```

Comment:
The equivalent is bento isn't it? See https://github.com/steveswinsburg/oracle12c-vagrant/blob/master/Vagrantfile